app/assets/stylesheets/components/page_specific/_link_list.scss

```diff
@@ -0,0 +1,42 @@
+/*
+  Link list component
+  Used on category pages to list out yellow underlined links
+*/
+.link-list {
+  @include body(22, 40);
+  font-weight: bold;
+
+  .l-category--image & {
+    @include respond-to($mq-m) {
+      width: 60%;
+    }
+  }
+}
+
+.link-list__item {
+  display: inline;
+}
+
+.link-list__item:before {
+  content: none !important;
+}
+
+.link-list__item:after {
+  content: ", ";
+}
+ 
+.link-list__item:last-child:after {
+  content: "";
+}
+
+.link-list__item__link {
+  color: $color-true-black !important;
```

Comment:
I think avoid `!important` here and just specify the link states so `:link, :visited` etc. That should get specificity right?